src/python_testing/matter_testing_infrastructure/matter/testing/commissioning.py

```diff
@@ -379,3 +379,268 @@ def qr_code(self):
     def manual_code(self):
         return SetupPayload().GenerateManualPairingCode(self.passcode, self.vendor_id, self.product_id, self.discriminator,
                                                         self.custom_flow, self.capabilities, self.version)
+
+
+# Commissioning Status Detection Functions
+
+async def _establish_pase_session(
```

Comment:
Can this all be switched over to use FindOrEstablishPASESession instead? This logic already exists.